code/obj/artifacts/artifact_objects/bombs.dm

```diff
@@ -290,3 +291,177 @@ ABSTRACT_TYPE(/datum/artifact/bomb)
 		SPAWN_DBG(recharge_delay)
 			if (O)
 				O.ArtifactDeactivated()
+
+// matter transmutation bomb
+
+/obj/machinery/artifact/bomb/transmute
+	name = "artifact matter transmutation bomb"
+	associated_datum = /datum/artifact/bomb/transmute
+
+	ArtifactHitWith(obj/item/O, mob/user)
+		if(..())
+			return
+		var/datum/artifact/bomb/transmute/A = src.artifact
+		O.setMaterial(A.mat)
+		boutput(user, "\The [O] turn\s into [A.mat.name]!")
+
+/datum/artifact/bomb/transmute
+	associated_object = /obj/machinery/artifact/bomb/transmute
+	validtypes = list("wizard","eldritch","martian","ancient","precursor")
+	rarity_class = 4
+	react_xray = list(17,95,95,3,"ANOMALOUS")
+	warning_initial = ""
+	warning_final = ""
+	alarm_initial = null
+	alarm_during = null
+	alarm_final = "sound/machines/satcrash.ogg"
+	var/material = "gold"
+	var/datum/material/mat = null
+	var/affects_organic = 0 // 1 means material human, 2 means material statue
+	var/range = 7
+	var/smoothEdge = 0 // with this, the area will be perfectly circular
+
+	post_setup()
+		affects_organic = pick(
+			prob(5); 0, // does nothing
+			prob(2); 1, // material human
+			prob(2); 2) // material statue
```

Comment:
Not too big of an issue but this might be cleaner if you `#define`d the three possibilities and used the defines for clarity. (Probably define them at the beginning of the file and undefine at the end.)

Reply:
Yeah, I was thinking that while writing it, but then I thought let's not bother, but you caught me.^^